posthog/management/commands/sync_persons_to_clickhouse.py

```diff
@@ -0,0 +1,151 @@
+from uuid import UUID
+
+import structlog
+from django.core.management.base import BaseCommand
+
+from posthog.client import sync_execute
+from posthog.models.person import PersonDistinctId
+from posthog.models.person.person import Person
+from posthog.models.person.util import _delete_ch_distinct_id, create_person, create_person_distinct_id
+
+logger = structlog.get_logger(__name__)
+
+PERSON_TABLE = "person"
+DISTINCT_ID_TABLE = "persondistinctid"
+
+
+class Command(BaseCommand):
+    help = """Sync person or distinct id tables from postgres to ClickHouse.
+        Lookup from Postgres and with a lower version in ClickHouse will be updated.
+        Note higher versions in ClickHouse will be ignored.
+        Recommended: run first without `--live-run` and first for person table, then distinct_id table
+        """
+
+    def add_arguments(self, parser):
+        parser.add_argument("--team-id", default=None, type=int, help="Specify a team to fix data for.")
+        parser.add_argument(
+            "--tables", default=[PERSON_TABLE, DISTINCT_ID_TABLE], action="append", help="Which tables to sync"
+        )
+        parser.add_argument(
+            "--deletes", action="store_true", help="process deletes for data in ClickHouse but not Postgres"
+        )
+        parser.add_argument("--live-run", action="store_true", help="Run changes, default is dry-run")
+
+    def handle(self, *args, **options):
+        run(options)
+
+
+def run(options):
+    live_run = options["live_run"]
+    deletes = options["deletes"]
+
+    if not options["team_id"]:
+        logger.error("You must specify --team-id to run this script")
+        exit(1)
+
+    if not set(options["tables"]) <= {PERSON_TABLE, DISTINCT_ID_TABLE}:
+        logger.error("Illegal value in --tables")
+        exit(1)
+
+    team_id = options["team_id"]
+
+    if PERSON_TABLE in options["tables"]:
+        run_person_sync(team_id, live_run, deletes)
+
+    if DISTINCT_ID_TABLE in options["tables"]:
+        run_distinct_id_sync(team_id, live_run, deletes)
+
+
+def run_person_sync(team_id: int, live_run: bool, deletes: bool):
+    logger.info("Running person table sync")
+    # lookup what needs to be updated in ClickHouse and send kafka messages for only those
+    persons = Person.objects.filter(team_id=team_id)
+    rows = sync_execute(
+        """
+            SELECT id, max(version) FROM person WHERE team_id = %(team)s GROUP BY id HAVING max(is_deleted) = 0
```

Comment:
nit: again, why `team` instead of `team_id`?

Reply:
was copy-pasta from somewhere